src/customEditors/EditorSensorItem.tsx

```diff
@@ -52,12 +52,19 @@ export const EditorSensorItem: React.FC<Props> = (props: Props) => {
       </Field>
       {/* </HorizontalGroup> */}
 
-      <Field label="Mapping ID" description="Past here the ID of the mapping you want to use for this sensor">
+      <Field
+        label="Mapping ID"
+        description="Paste here comma-separated IDs of the mappings you want to use for this sensor"
```

Comment:
I think that pasting values and not being able to write them is bad for the users, I was confused too
I think this could be a solution: [tags input](https://developers.grafana.com/ui/latest/index.html?path=/story/forms-tagsinput--basic)
what do you think?

Reply:
```
<Label description="mapping ids list">
  Mapping IDs
</Label>
<TagsInput
  // @ts-ignore - will be removed when upgrading grafana/ui
  placeholder="Add a new mapping"
  tags={sensor.mappingIds}
  onChange={mappingIds => {
    updateSensor(sensor => {
      sensor.mappingIds = mappingIds;
    });
  }}
/>
```